app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -34,7 +34,7 @@
 import com.firebase.ui.auth.AuthUI;
 import com.firebase.ui.auth.AuthUI.IdpConfig;
 import com.firebase.ui.auth.IdpResponse;
-import com.firebase.ui.auth.ui.ResultCodes;
+import com.firebase.ui.auth.ResultCodes;
```

Comment:
I think this would technically require a major version (if you're going by semver) since old code would break when updated.

Could we keep the `.ui.ResultCodes` class but have it be all `@Deprecated` constants that are just symlinks to the new class?  Then it's fully backwards compatible.